sn_node/src/node/messaging/node_msgs.rs

```diff
@@ -302,17 +269,16 @@ impl MyNode {
                         }
 
                         trace!(
-                            "=========>> This node has been approved to join the network at {:?}!",
+                            "=========>> This node has been approved to join the section at {:?}!",
                             target_sap.prefix(),
                         );
 
-                        debug!("[NODE READ]: relocation_state read");
-                        let relocate_state_present = node.read().await.relocate_state.is_some();
-                        debug!("[NODE READ]: relocation_state read got");
-                        if relocate_state_present {
-                            let mut node = node.write().await;
-                            debug!("[NODE WRITE]: relocation_state write got...");
-                            node.relocate_state = None;
+                        if decision
+                            .proposals
+                            .keys()
+                            .filter(|n| n.state() == MembershipState::Joined)
+                            .all(|n| n.previous_name().is_some())
```

Comment:
Do you mean `any()` here? As in, if there was a node we accepted who had a previous name, then there was a relocation that completed.

```suggestion
                            .any(|n| n.previous_name().is_some())
```

Reply:
Yeah IDK, I reused that bit of the code.

Reply:
Yeah, it's from ln 265.